src/puppetlabs/puppetdb/cli/services.clj

```diff
@@ -42,33 +42,34 @@
      data may linger in the database. We periodically sweep the
      database, compacting it and performing regular cleanup so we can
      maintain acceptable performance."
-  (:require [puppetlabs.puppetdb.scf.storage :as scf-store]
-            [puppetlabs.puppetdb.command.dlo :as dlo]
-            [puppetlabs.puppetdb.query.population :as pop]
-            [puppetlabs.puppetdb.jdbc :as pl-jdbc]
-            [puppetlabs.puppetdb.mq :as mq]
+  (:require [clj-time.core :refer [ago]]
+            [clojure.java.io :as io]
             [clojure.java.jdbc :as sql]
             [clojure.tools.logging :as log]
-            [puppetlabs.puppetdb.http.server :as server]
-            [puppetlabs.puppetdb.config :as conf]
-            [puppetlabs.kitchensink.core :as kitchensink]
-            [robert.hooke :as rh]
-            [puppetlabs.trapperkeeper.core :refer [defservice] :as tk]
-            [puppetlabs.trapperkeeper.services :refer [service-id service-context]]
             [compojure.core :as compojure]
-            [clojure.java.io :refer [file]]
-            [clj-time.core :refer [ago]]
+            [me.raynes.fs :as fs]
             [overtone.at-at :refer [mk-pool interspaced]]
-            [slingshot.slingshot :refer [throw+ try+]]
-            [puppetlabs.puppetdb.time :refer [to-seconds to-millis parse-period
-                                              format-period period?]]
+            [puppetlabs.kitchensink.core :as kitchensink]
+            [puppetlabs.puppetdb.cheshire :as json]
+            [puppetlabs.puppetdb.command.constants :refer [command-names]]
+            [puppetlabs.puppetdb.command.dlo :as dlo]
+            [puppetlabs.puppetdb.config :as conf]
+            [puppetlabs.puppetdb.http.server :as server]
+            [puppetlabs.puppetdb.jdbc :as pl-jdbc]
             [puppetlabs.puppetdb.jdbc :refer [with-transacted-connection]]
-            [puppetlabs.puppetdb.scf.migrate :refer [migrate! indexes!]]
             [puppetlabs.puppetdb.meta.version :refer [version update-info]]
-            [puppetlabs.puppetdb.command.constants :refer [command-names]]
-            [puppetlabs.puppetdb.cheshire :as json]
+            [puppetlabs.puppetdb.mq :as mq]
             [puppetlabs.puppetdb.query-eng :as qeng]
-            [puppetlabs.puppetdb.utils :as utils])
+            [puppetlabs.puppetdb.query.population :as pop]
+            [puppetlabs.puppetdb.scf.migrate :refer [migrate! indexes!]]
+            [puppetlabs.puppetdb.scf.storage :as scf-store]
+            [puppetlabs.puppetdb.time :refer [to-seconds to-millis parse-period
+                                              format-period period?]]
+            [puppetlabs.puppetdb.utils :as utils]
+            [puppetlabs.trapperkeeper.core :refer [defservice] :as tk]
+            [puppetlabs.trapperkeeper.services :refer [service-id service-context]]
+            [robert.hooke :as rh]
+            [slingshot.slingshot :refer [throw+ try+]])
```

Comment:
Sorry about the namespace changes above ^ honestly I'm not sure what happened there, I don't remember moving things around. I'm sure I must have had a good reason >.>